Source/appfat.cpp

```diff
@@ -29,7 +29,7 @@ void MsgBox(const char *pszFmt, va_list va)
 {
 	char text[256];
 
-	vsnprintf(text, 256, pszFmt, va);
+	vsnprintf(text, sizeof(text), pszFmt, va);

```

Comment:
Why not use a `std::string` directly?

Reply:
Ah, I suppose this would be out of scope for this PR?

Reply:
Just did it quickly to avoid some warnings when trying out CLion :)

Hopefully, this makes some tools better able to reason about this code, and maybe it will even make refactoring easier :o